packages/components/psammead-podcast-promo/src/index.test.jsx

```diff
@@ -85,28 +85,13 @@ describe('Podcast Promo', () => {
   });
 
   describe('card-call-to-action', () => {
-    assertTypeOfElement(PodcastPromo.Card.CallToAction, 'span');
+    assertTypeOfElement(PodcastPromo.Card.CallToAction, 'p');
     shouldMatchSnapshot(
       'should match snapshot',
       <PodcastPromo.Card.CallToAction script={scripts.latin} service="russian">
         Click Me
       </PodcastPromo.Card.CallToAction>,
     );
-    it('should be aria-hidden', () => {
```

Comment:
Just needs a README update - it has "This component is `aria-hidden` to screen readers." in the docs for this component

Reply:
👍